index.js

```diff
@@ -0,0 +1,132 @@
+let interactedContainer = document.getElementById("interactedContainer")
+let interactionContainer = document.getElementById("interactionContainer")
+let userhandsimage = document.getElementById("userhandsImage")
+let computerhandsimage = document.getElementById("computerHandsImage")
+let gamestate = document.getElementById("gameState")
+let gamestatemobile = document.getElementById("gameStateMobile")
+let rules = document.getElementById("rulesPicture")
+let rulescontainer = document.getElementById("rulesContainer")
+let clickPaper = document.querySelector(".paper")
+let clickRock = document.querySelector(".rock")
+let clickScissors = document.querySelector(".Scissors")
+let rulesPop = document.getElementById("rules")
+let playagain = document.querySelectorAll(".playagain")
+clickPaper.addEventListener("click",paper)
+clickRock.addEventListener("click",rock)
+clickScissors.addEventListener("click",scissors)
+rulesPop.addEventListener("click",rulesPopup)
+playagain.forEach(button => {
+    button.addEventListener("click",playAgain)
+});
+let score = 0
+function randomNumber(){
+    let random = Math.floor(Math.random() * 3) + 1
+    if(random ===1){
+        computerhandsimage = document.getElementById("computerHandsImage").src = "assets/Rock.png"   
```

Comment:
```suggestion
        computerhandsimage.src = "assets/Rock.png"   
```
From L4 above, `computerhandsimage` should already reference the target DOM element. As such, you don't need to repeat the DOM lookup.

Same with L28, L31 below

Reply:
I always thought I had to made reference to the DOM everytime I needed to use it
This is a good fix.
I've taken note, thanks